File: bin/Debug/net5.0/v4.4.2/lgcy-on-lgcy/examples/t_api-captcha-mvc5-aspnet.identity/csharp/App_Code/CaptchaHelper.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

using BotDetect.Web.Mvc;

namespace Mvc50CaptchaExampleCSharp.App_Code
{
    public static class CaptchaHelper
    {
        public static MvcCaptcha GetRegistrationCaptcha()
        {
            // create the control instance
            MvcCaptcha registrationCaptcha = new MvcCaptcha("RegistrationCaptcha");

            // set up client-side processing of the Captcha code input textbox
            registrationCaptcha.UserInputID = "CaptchaCode";

            // Captcha settings
            registrationCaptcha.ImageSize = new System.Drawing.Size(255, 50);

            return registrationCaptcha;
        }

        public static MvcCaptcha GetLoginCaptcha()
        {
            // create the control instance
            MvcCaptcha registrationCaptcha = new MvcCaptcha("LoginCaptcha");

            // set up client-side processing of the Captcha code input textbox
            registrationCaptcha.UserInputID = "CaptchaCode";

            // Captcha settings
            registrationCaptcha.ImageSize = new System.Drawing.Size(255, 50);

            return registrationCaptcha;
        }

        public static MvcCaptcha GetChangePasswordCaptcha()
        {
            // create the control instance
            MvcCaptcha registrationCaptcha = new MvcCaptcha("ChangePasswordCaptcha");

            // set up client-side processing of the Captcha code input textbox
            registrationCaptcha.UserInputID = "CaptchaCode";

            // Captcha settings
            registrationCaptcha.ImageSize = new System.Drawing.Size(255, 50);

            return registrationCaptcha;
        }
    }
}

File: bin/Debug/net5.0/v4.4.2/lgcy-on-lgcy/examples/t_api-captcha-mvc5-aspnet.identity/vbnet/App_Code/CaptchaHelper.vb
Imports BotDetect.Web.Mvc
Public Class CaptchaHelper

    Public Shared Function GetRegistrationCaptcha() As MvcCaptcha

        ' create the control instance
        Dim registrationCaptcha As MvcCaptcha = New MvcCaptcha("RegistrationCaptcha")

        ' set up client-side processing of the Captcha code input textbox
        registrationCaptcha.UserInputID = "CaptchaCode"

        ' Captcha settings
        registrationCaptcha.ImageSize = New System.Drawing.Size(255, 50)

        Return registrationCaptcha

    End Function

    Public Shared Function GetLoginCaptcha() As MvcCaptcha

        ' create the control instance
        Dim registrationCaptcha As MvcCaptcha = New MvcCaptcha("LoginCaptcha")

        ' set up client-side processing of the Captcha code input textbox
        registrationCaptcha.UserInputID = "CaptchaCode"

        ' Captcha settings
        registrationCaptcha.ImageSize = New System.Drawing.Size(255, 50)

        Return registrationCaptcha

    End Function

    Public Shared Function GetChangePasswordCaptcha() As MvcCaptcha

        ' create the control instance
        Dim registrationCaptcha As MvcCaptcha = New MvcCaptcha("ChangePasswordCaptcha")

        ' set up client-side processing of the Captcha code input textbox
        registrationCaptcha.UserInputID = "CaptchaCode"

        ' Captcha settings
        registrationCaptcha.ImageSize = New System.Drawing.Size(255, 50)

        Return registrationCaptcha

    End Function

End Class
